docs/python/linting.md

```diff
@@ -14,263 +14,63 @@ Linting highlights syntactical and stylistic problems in your Python source code
+| importStrategy | `useBundled` | `useBundled` means extension uses bundled version of the linter. `fromEnvironment` means it will attempt to load from your selected python environment first, otherwise falls back to bundled version. |
 
-### Message category mapping
+## Disabling Linter
 
-The Python extension maps all Prospector errors and warnings to the Error (E) category.
+Linters, if installed, are enabled by default. You can disable them by disabling the extension per workspace.
 
-## Flake8
 
-### Command-line arguments and configuration files
-
-See [Invoking Flake8](https://flake8.pycqa.org/en/latest/user/invocation.html) for general switches. For example, to ignore error E303 (too many blank lines), use the following setting:
-
-```json
-"python.linting.flake8Args": ["--ignore=E303"]
-```
-
-By default, Flake8 ignores E121, E123, E126, E226, E24, and E704.
-
-Flake8 user options are read from the `C:\Users\<username>\.flake8` (Windows) or `~/.config/flake8` (macOS/Linux) file.
+## Run linting
 
-At the project level, options are read from the `[flake8]` section of a `tox.ini`, `setup.cfg`, or `.flake8` file.
+Linting will automatically run on any file on Open or Save. It will clear out the problems for a file on Close.
 
-For details, see [Flake8 configuration](https://flake8.pycqa.org/en/latest/user/configuration.html).
+Issues are shown in the **Problems** panel and as wavy underlines in the code editor. Hovering over an underlined issue displays the details:
 
-### Message category mapping
+![Linting messages in the editor and the Problems panel](images/linting/lint-messages.png)
 
-The Python extension maps flake8 message categories to VS Code categories through the following settings. If desired, change the setting to change the mapping.
 
-| Flake8 category | Applicable setting<br/>(python.linting.) | VS Code category mapping |
-| --- | --- | --- |
-| F | flake8CategorySeverity.F | Error |
-| E | flake8CategorySeverity.E | Error |
-| W | flake8CategorySeverity.W | Warning |
+## Code Actions
 
-## mypy
+Some linters may offer code actions on problems, which can halp address the issue.
 
-### Message category mapping
+## Logging
 
-The Python extension maps mypy message categories to VS Code categories through the following settings. If desired, change the setting to change the mapping.
+Logs for linters should be available in the `Output` tab. Select `<linter name>` from the drop down.
 
-| mypy category | Applicable setting<br/>(python.linting.) | VS Code category mapping |
-| --- | --- | --- |
-| error | mypyCategorySeverity.error | Error |
-| note | mypyCategorySeverity.note | Information |
+You can change the log level for a linter extension by running the command pallet commmand `Developer: Set Log Level` select the extension from the entension logs group. and change the level.
 
 ## Troubleshooting linting
```

Comment:
Should we remove this troubleshooting section? It feels heavily applied to the built-in functionality

Reply:
This was one of the items I wanted to discuss. I think we need to add items here that belong to the extension cases.

Reply:
"LanguageServer crashes 5 times" this case should not happen any more. I have updated the Trouble shooting with the most common issue I see with linting.